benchmarks/NodeClassification/models/gin.py

```diff
@@ -0,0 +1,84 @@
+"""
+GIN model in GLI.
+
+References:
+https://github.com/dmlc/dgl/tree/master/examples/pytorch/gin
+"""
+
+from torch import nn
+import torch.nn.functional as F
+
+from dgl.nn.pytorch.conv import GINConv
+from dgl.nn.pytorch.glob import SumPooling
+
+
+class MLP(nn.Module):
+    """Construct two-layer MLP-type aggreator for GIN model."""
+
+    def __init__(self, input_dim, hidden_dim, output_dim):
+        """Init."""
+        super().__init__()
+        self.linears = nn.ModuleList()
+        # two-layer MLP
+        self.linears.append(nn.Linear(input_dim, hidden_dim, bias=False))
+        self.linears.append(nn.Linear(hidden_dim, output_dim, bias=False))
+        self.batch_norm = nn.BatchNorm1d((hidden_dim))
+
+    def forward(self, x):
+        """Forward."""
+        h = x
+        h = F.relu(self.batch_norm(self.linears[0](h)))
+        return self.linears[1](h)
+
+
+class GIN(nn.Module):
+    """GIN network."""
+
+    def __init__(self, g, input_dim, hidden_dim, output_dim):
```

Comment:
I'm wondering if we should pass parameters of GIN layers and MLP layers instead of hardcoding them.

Reply:
Great suggestion!